Readme.md

```diff
@@ -161,3 +161,6 @@ script:
 
 ```
 
+## Known Issue
```

Comment:
Lets use `###` and put it under the [Private services...](https://github.com/SymfonyTest/symfony-bundle-test#private-services-in-symfony-4)